docs/nodes/nodes-library/nodes/GoogleSheets/README.md

```diff
@@ -34,39 +34,137 @@ The final workflow should look like the following image.
 
 The Start node exists by default when you create a new workflow.
 
-
+::: v-pre
 ### 2. Set node
 
-1. Click on the ***Add Value*** button and select 'Number' from the dropdown list.
-2. Enter `id` in the ***Name*** field.
-3. Click on the ***Add Value*** button and select 'String' from the dropdown list.
-4. Enter `name` in the ***Name*** field.
-5. Enter the value for the name in the ***Value*** field.
-6. Click on ***Execute Node*** to run the node.
-
-![Using the Set node to set data to be inserted by the Google Sheets node](./Set_node.png)
+The example workflow uses the Set node to generate data that we want to add to the Google Sheet. We can use other nodes, like [HTTP Request node](../../core-nodes/HTTPRequest/README.md), to get the data that we want to add to the Google Sheet.
+
+1. Click on the ***Add Value*** button and select 'String' from the dropdown list.
+2. Enter `Name` in the ***Name*** field.
+3. Enter the name of the house in the ***Value*** field.
+4. Click on the ***Add Value*** button and select 'String' from the dropdown list.
+5. Enter `Rent` in the ***Name*** field.
+6. Enter the rent of the house with the currency (example $1000) in the ***Value*** field.
+7. Click on the ***Add Value*** button and select 'String' from the dropdown list.
+8. Enter `City` in the ***Name*** field.
+9. Enter the name of the city in the ***Value*** field.
+10. Click on the ***Add Value*** button and select 'Number' from the dropdown list.
+11. Enter `id` in the ***Name*** field.
+12. Click on the gears icon next to the ***Value*** field click on ***Add Expression***.
+13. Paste the following expression `{{Math.floor(Math.random()*1000)}}`. This expression will generate a three-digit random number.
+14. Toggle ***Keep Only Set*** to true.
+15. Click on ***Execute Node*** to run the node.
+:::
 
+![Using the Set node to set data to be inserted in the Google Sheets node](./Set_node.png)
 
+  
 ### 3. Google Sheets node (Append)
 
+This node will append the data in the Google Sheet.
+
 1. First of all, you'll have to enter credentials for the Google Sheets node. You can find out how to do that [here](../../../credentials/Google/README.md), in the section 'Google Drive / Sheets API'.
```

Comment:
This needs to be updated. There's no 'Google Drive / Sheets API' section in the linked document.